### PR TITLE
Anniversary cheatsheet

### PR DESCRIPTION
**Pull request by JerbiAhmed**
# Goodie Cheat Sheet Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**

Display a Cheatsheet for latin and common anniversaries names
**What is the data source for your Instant Answer? (Provide a link if possible)**

https://en.wikipedia.org/wiki/Anniversary#Latin-derived_numerical_names
**Why did you choose this data source?**

Complete and fairly trusted.
**Are there any other alternative (better) data sources?**

No.
**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

History ,Literature ,everybody.
**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

Yes.
**Are you having any problems? Do you need our help with anything?**

No.
**Where did you hear about DuckDuckHack? (For first time contributors)**

journaldunet.com Website 
**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
image

-----
IA Page: https://duck.co/ia/view/anniversary_cheat_sheet

### CODE REVIEW

**Comment by zekiel** [comment]
thanks for contributing @JerbiAhmed  We'll take a look at this asap and get back to you with feedback and next steps!

File: share/goodie/cheat_sheets/anniversary.json [hunk 1]
```diff
@@ -0,0 +1,211 @@
+{
+"id": "anniversary_cheat_sheet",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/anniversary.json]: Can you fix this indentation here?

**Comment by mintsoft** [comment]
@JerbiAhmed This looks good to me bar the minor indentation issue. Although I think it would be better if the common and latin were aligned for the same year (similiar to how it is on the source). Can you add an N/A to the 6 month for Common and then it'll line up?

**Comment by mintsoft** [comment]
@abeyang any additional thoughts?

File: share/goodie/cheat_sheets/anniversary.json [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    "name": "Anniversary",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/anniversary.json]: Let's change this to "Anniversary Names"

File: share/goodie/cheat_sheets/anniversary.json [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+            "val": "100 years",
+            "key": "Platinum / Centenary"
+        },{
+            "val": "200 years",
+            "key": "Bicentenary"
+        }, {
+            "val": "250 years",
+            "key": "Quarter-millennial"
+        }, {
+            "val": "300 years",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/anniversary.json]: Looks like this list keeps going to 800 on Wikipedia? https://en.wikipedia.org/wiki/Anniversary#Latin-derived_numerical_names

File: share/goodie/cheat_sheets/anniversary.json [hunk 4]
```diff
@@ -0,0 +1,211 @@
+{
+"id": "anniversary_cheat_sheet",
+    "name": "Anniversary",
+    "description": "Anniversaries special names",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/anniversary.json]: Let's make this a little more *descriptive*: "Common and Latin terms for anniversaries from 6 months up to 2000 years"

**Comment by JerbiAhmed** [comment]
@mintsoft @moollaza Thank you for the reviews .. Updated.

**Comment by abeyang** [comment]
@JerbiAhmed looking good. Can we use proper title casing? So it should be `Diamond of Monarchs`, for example. Also, few questions:

1. Are these terms only used for marriage anniversaries? That was my understanding...
2. If so, then anything beyond 200 years seem a bit overboard, no?

**Comment by JerbiAhmed** [comment]
@abeyang Indeed those terms are usually used for marriages anniversaries but not only.
It may refers to any other event here are some examples how those terms can be used: 
-Magna Carta Octocentennial (800 years since the agreement of the Great Charter).
-Quincentennial of the American People ( marks 500 years of the arrival of the Western world to the eastern shores of the United States).
-Semiseptcentennial of Dartmouth (the 350th anniversary since the incorporation of the town of Dartmouth)...

**Comment by abeyang** [comment]
@JerbiAhmed Ok, you're right. :smile: I'm fine with it, then.

**Comment by JerbiAhmed** [comment]
Thank you :)
Title casing fixed.

**Comment by moollaza** [comment]
LGTM :+1:

**Comment by moollaza** [comment]
@JerbiAhmed congrats! It's live: https://duckduckgo.com/?q=anniversary+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**